inference/colorization_pipeline.py

```diff
@@ -54,12 +53,13 @@ def __init__(self, model_path, input_size=256, model_size='large'):
         self.model.eval()
 
     @torch.no_grad()
-    def process(self, img):
-        self.height, self.width = img.shape[:2]
-        # print(self.width, self.height)
-        # if self.width * self.height < 100000:
-        #     self.input_size = 256
+    def process(self, img_path):
+        img = cv2.imread(img_path)
+        if img is None:
+            print(f"Error: Unable to load image '{img_path}'. Skipping...")
+            return None
```

Comment:
I wonder if the change here is necessary because it will also affect ImageColorizationPipelineHF class.

https://github.com/piddnad/DDColor/blob/5a764a218288ad059b5bf281fca11847c8838fe3/inference/colorization_pipeline_hf.py#L17.

Reply:
Hi @gessyoo, thank you for your recognition of our project! 
I understand what you're trying to do, but I think it's better to keep the process() method the same behavior as before. As for processing image directories and saving them, maybe we can write these codes outside of the class, or to write a separate class method (such as process_img_folder() or something). 
We do observe that some results can be oversaturated, so adding some color adjustment parameters is a good idea!

Reply:
I've added some experimental color correction options with defaults to produce a hopefully more natural, less saturated output.  I've also updated the teaser.png to show the color correction effect.